### ISSUE TITLE
Creating new patient - validation messages not appropriate

### ISSUE THREAD

**Issue by savanrajkotiya** [HospitalRun/hospitalrun-frontend#2699]
Before you submit an issue we recommend you drop into our [Slack workspace](https://hospitalrun-slack.herokuapp.com/) and ask any questions you have or mention any problems you've had getting started with HospitalRun.

**Please read this entire template before posting any issue. If you ignore these instructions
and post an issue here that does not follow the instructions, your issue might be closed,
locked, and assigned the `missing discussion` label.**

## 🐛 Bug Report

The validation messages of inputs are not appropriate
ex. 'Must be a valid phone number.' but what's a valid phone number pattern.?
PFA the Image

## To Reproduce

- Try creating a new patient with contact number digits more than 14-15 digits

## Expected behavior

A clear and appropriate validation message should appear so that user will get to know what mistake that they made.

## Your Environment

- *node version*: 10
- *fastify version*: >=1.0.0
- *os*:  Ubuntu 20.0.4
- *any other relevant information*

![Screenshot from 2021-06-14 21-52-31](https://user-images.githubusercontent.com/37897785/121927455-af7fb680-cd5c-11eb-935c-a64ee6afb290.png)

**Comment by dangnhon** [comment]
Hi there @tehKapa, what would be the expected validation message? Something on the lines of "Must be valid 10 digit" or etc? I'm interested in this issue.

**Comment by tehKapa** [comment]
@dangnhon yes, it would be necessary to do a validation in terms of country code and max phone digits

**Comment by reginz** [comment]
Hi @tehKapa. So I found this https://github.com/jackocnr/intl-tel-input I think you can use this input field and maybe you can check if user input is equal to selectedCountryPlaceholder or not.

**Comment by tehKapa** [comment]
@reginz looks fine

**Comment by abhiFSD** [comment]
hello @tehKapa can I work on this issue?
